pages/common/sshd.md

```diff
@@ -0,0 +1,21 @@
+# sshd
+
+> Secure Shell Daemon - allows remote machines to securely log in to the current machine.
+> Remote machines can execute commands as it is executed at this machine.
+> More information: <https://linux.die.net/man/8/sshd>.
```

Comment:
I think we should use the official manual:
```suggestion
> More information: <https://man.openbsd.org/sshd>.
```

Reply:
Thanks

Reply:
@nobiit but you didn't change the link.

Reply:
Oh. I forgot, thank you again